promscale/installation/kubernetes/tobs.md

```diff
@@ -0,0 +1,126 @@
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/installation/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The Observability Stack, is a pre-packaged distribution of observability tools
+and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+such as Prometheus, Grafana, Jaeager, Promlens, TimescaleDB, and others.
+Together, these provide a maintainable solution to analyze the traffic on the
+server and identify any potential problems with a deployment. You can use Helm
+charts to configure and update [tobs][tobs] deployments. 
+
+## Before you begin
+*  Understand the [resource requirements][resource-requirements] for your cluster.
+*  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
+*  Install [kubectl][kubectl] in your local environment and
+   [connect to your cluster][connect-to-cluster].
+*  Install [Helm 3][helm] in your local machine to add and update the repositories:
+   1.  Add the stable Helm charts repository to your Helm repositories.
+    ```bash
+    helm repo add stable https://charts.helm.sh/stable
+    ```
+   1.  Update your Helm repositories.
+    ```bash
+    helm repo update
+    ```
+
+
+## Installing Promscale with tobs 
+You can deploy tobs on your cluster and access it locally with `kubectl`
+[Port-Forward][port-forward].
+
+<procedure>
+
+1.  Create the `observability` namespace on your cluster:
+    ```bash
+    kubectl create namespace observability
+    ```   
+1. Install a certificate manager for your cluster:
+   ```bash
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
+   ```
+1.  Using Helm, deploy the TOBS release labeled `tobs` in the `observability`
+    namespace on your cluster:
+    ```bash
+    helm repo add timescale https://charts.timescale.com/
+    helm repo update
+    helm install --wait tobs timescale/tobs --namespace observability
+    ```
+1.  Verify that the Prometheus Operator has been deployed to your cluster and
+    its components are running and ready by checking the pods in the
+    `observability` namespace:
+    ```bash
+    kubectl -n observability get pods
+    ```
+    
+</procedure>
+
+### Accessing the interfaces with port-forward
+
+<procedure>
+
+1.  List the services running in the `observability` namespace and review their
+    respective ports:
+    ```bash
+    kubectl -n observability get svc
+    ```
+    You can use `kubectl` [port-forward][port-forward] to open a connection to a service, then access the interface of the service by entering the corresponding ports in your web browser.
```

Comment:
Move this to above the code block, or into its own step. Also needs a line break.